Packs/ArcherRSA/Integrations/ArcherV2/ArcherV2.py

```diff
@@ -32,6 +32,8 @@
 
 ACCOUNT_STATUS_DICT = {1: 'Active', 2: 'Inactive', 3: 'Locked'}
 
+API_ENDPOINT = demisto.params().get('api_endpoint', 'api')
```

Comment:
* move to the main.
* use the following implementation of default value:
`params.get('api_endpoint') or 'rsaarcher/api'`